mdm/api/src/main/java/com/gentics/mesh/core/data/dao/MicroschemaDao.java

```diff
@@ -91,22 +95,55 @@ default HibMicroschemaVersion fromReference(HibProject project, MicroschemaRefer
 
 	/**
 	 * Add the microschema to the project.
-	 * 
+	 *
 	 * @param project
 	 * @param user
 	 * @param microschemaContainer
 	 * @param batch
 	 */
-	void addMicroschema(HibProject project, HibUser user, HibMicroschema microschemaContainer, EventQueueBatch batch);
+	default void addMicroschema(HibProject project, HibUser user, HibMicroschema microschemaContainer, EventQueueBatch batch) {
```

Comment:
Code was removed by @plyhun 
I've also removed `MicroschemaContainerVersionImpl.delete(ac)` since I couldn't find any direct usage of it and the functionality is now implemented by `MicroschemaDao.deleteVersion(ac)`. 
@plyhun is that correct? I saw there was a comment from you, I hope it's no longer relevant, I couldn't find any direct usage of the method anymore